CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+Version 0.9.8
```

Comment:
With the changes in this PR I would also bump the minor version, not only patch.